Resources/Private/CodeTemplates/Extbase/gitignore.t

```diff
@@ -0,0 +1,2 @@
+/.idea/
```

Comment:
I disagree about that. This is IDE specific, this should go into your **local** `.git/info/exclude` file.